### ISSUE TITLE
Internal "Form Name" for form pages

### ISSUE THREAD

**Issue by thenewguy** [coderedcorp/coderedcms#455]
It would be handy if form pages had an name associated with them that was only for reference in the editor and file exports.  The page title tends to be similar (and difficult to differentiate) when you have different "subscribe" forms for tracking the effectiveness of different affiliate links.  It would be useful if this was displayed on the "/admin/forms/" listing as well as in the exported files.

**Comment by vsalvino** [comment]
Good idea, I think the URL would suffice here (perhaps shown smaller below the form name).

Form submissions are directly from wagtail via `wagtail.contrib.forms`, so that would be the place to look if you're interested in making any changes.
